third_party/2and3/redis/client.pyi

```diff
@@ -307,8 +307,8 @@ class PubSub:
     def subscribe(self, *args: Text, **kwargs: Callable[[Any], None]) -> None: ...
     def unsubscribe(self, *args: Text) -> None: ...
     def listen(self): ...
-    def get_message(self, ignore_subscribe_messages: bool = ..., timeout: int = ...) -> Optional[bytes]: ...
-    def handle_message(self, response, ignore_subscribe_messages=...): ...
+    def get_message(self, ignore_subscribe_messages: bool = ..., timeout: float = ...) -> Optional[Dict[str, Any]]: ...
+    def handle_message(self, response, ignore_subscribe_messages: bool = ...) -> Dict[str, Any]: ...
```

Comment:
The return type should be `Optional[]`

Reply:
Oooops! Thanks for catching that.

I've amended the incorrect commit and rebased onto latest master. See https://github.com/python/typeshed/pull/3408/commits/310b31841724c523c60c1e7e6dfe28b9d721586f for the amended commit.